### DISCUSSION TITLE
How do you use pico with an embedded editor?

### DISCUSSION THREAD

**Discussion by nibl** [picocss/pico#307] [Help] [answered]
### Is there a way to exclude an element from pico?

I tried pico with easyMDE editor and it's no longer usable. See screenshots.

The first image shows how the editor should look (light mode).
![FD102D03-5F95-4D1A-BE5A-C6CA4AFE0991](https://user-images.githubusercontent.com/462866/215500486-0ad34ca6-4bd4-4487-be95-5241517f9d37.jpeg)

In the second image, the toolbar is stacked vertically and selecting a button creates a white strip. The text preview is very light grey and not readable. Text should be black. The screenshot uses dark mode because in light mode the toolbar is not visible.

![DA7DB41C-F312-4114-8FC6-2B85FAD4E505](https://user-images.githubusercontent.com/462866/215500494-3cad4947-cd7a-4269-8f9e-ff6f86f12530.jpeg)

**Comment by lucaslarroche** [comment, marked as answer]
As Pico affects all the HTML tags, you probably have to override the styles for the text editor.
`width: auto` for the editor's buttons, and `data-theme="light"` for the editable section, and probably a lot more.
There is no easy solution in this case.

**Reply by nibl** [reply]
Thanks, I’ll try your suggestions and see how far it’s possible to prevent pico influencing the embedded editor.
